### PR TITLE
color pallette updated

### PR DESCRIPTION
**Pull request by mgolwalla**

### CODE REVIEW

File: pages/sections/landing.tsx [hunk 1]
```diff
@@ -34,8 +34,8 @@ export default function Landing() {
 
   return (
     <>
-      <section className="relative flex flex-col-reverse w-full h-screen md:min-h-[65rem] my-0 md:mb-20 gap-6">
-        <div className="absolute left-1/2 -translate-x-1/2 w-full top-[30%] md:top-[25%] lg:top-[16%] flex flex-col justify-center items-center z-10 px-4">
+      <section className="relative flex flex-col-reverse w-full h-screen md:min-h-[60rem] mt-4 gap-6">
```
Comment (minsooerickim) [inline pages/sections/landing.tsx]: for the margins, im not sure if it looked different on ur laptop but on my the new changes look too cramped?

this is how it looks with ur code
image

and this is how it used to look with the old code
image

I personally think the old margins look better but what do you think?

**Review by minsooerickim — CHANGES_REQUESTED** [review]
what a beautiful color pallette 💯

File: pages/sections/landing.tsx (hunk 1 above)
Comment (mgolwalla) [reply pages/sections/landing.tsx]: lol star pupil mika told me to make those changes, ill revert them

Comment (msclal) [reply pages/sections/landing.tsx]: pls ok the margins were gonna get fixed in my next PR ✋🏻

Comment (minsooerickim) [reply pages/sections/landing.tsx]: Oo yea that woulda been fine too but looks like the director alr fixed it 🙌

**Pull request merged**